[all hosts] (manifest) Expand lifetime attribute description (#3324)
OfficeDev/office-js-docs-pr: docs/reference/manifest/runtime.md

|  Attribute  |  Required  |  Description  |
|:-----|:-----|:-----|
|  **resid**  |  Yes  | Specifies the URL location of the HTML page for your add-in. The `resid` can be no more than 32 characters and must match an `id` attribute of a `Url` element in the `Resources` element. |
|  [lifetime](#lifetime-attribute)  |  No  | The default value for `lifetime` is `short` and doesn't need to be specified. Outlook add-ins use only the `short` value. If you want to use a shared runtime in an Excel add-in, explicitly set the value to `long`. |

### lifetime attribute

Optional. Represents the length of time the add-in is allowed to run.

**Available values**

> `short`: Default. After the add-in is activated, it will be allowed to run for a maximum amount of time as specified by the platform. Currently, that's around 5 minutes. This is the only value supported by Outlook.

Reviewer: ```suggestion
`short`: Default. Used only for Outlook add-ins. After the add-in is activated, it will run for a maximum amount of time as specified by the platform. Currently, that's around 5 minutes. This is the only value supported by Outlook.
```

> `long`: The add-in will be allowed to run without any time limitations. This is the only value supported by a [shared runtime](../../develop/configure-your-add-in-to-use-a-shared-runtime.md).

Reviewer: ```suggestion
`long`: Used only when configuring a [shared JavaScript runtime](../../develop/configure-your-add-in-to-use-a-shared-runtime.md). The add-in can start on document open and run indefinitely. For example, task pane code will continue running even when the user closes the task pane. This is the only value supported by the shared runtime.
```
